src/iperf_api.c

```diff
@@ -953,11 +954,17 @@ iperf_on_connect(struct iperf_test *test)
     struct sockaddr_in *sa_inP;
     struct sockaddr_in6 *sa_in6P;
     socklen_t len;
+    struct iperf_time now;
+    time_t now_secs;
```

Comment:
This local variable is already declared above.

Reply:
yikes, forgot to add that change to the commit; fixed now